store/src/test/java/org/apache/rocketmq/store/StoreStatsServiceTest.java

```diff
@@ -22,9 +22,18 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.concurrent.atomic.AtomicReference;
 import org.junit.Test;
+import static org.junit.Assert.*;
 
 public class StoreStatsServiceTest {
 
+    @Test
+    public void StoreStatsServiceStringFromat() throws Exception {
+
+        StoreStatsService storeStatsService = new StoreStatsService();
+        new Thread(storeStatsService).start();
```

Comment:
wondering if it is necessary to start a new thread ~ will toString() fail if this service is not started ?

Reply:
if do not start a new thread，may collect nothing